### ISSUE TITLE
Regression: GTA IV and EFLC create and delete save files instantly and can't get ingame (#3005 onwards)

### ISSUE THREAD

**Issue by MarioSonic2987** [RPCS3/rpcs3#3585]
With https://github.com/RPCS3/rpcs3/commit/e8bde583ef2db0d9ad4f59a1c0647bbea10c9bc7, GTA IV and EFLC hang in loading screen. But in `dev_hdd0\home\00000001\savedata\BLES00229PROFILE` both games create and delete save files instantly.
Previously, on July, it went ingame.
![image](https://user-images.githubusercontent.com/20027681/31482731-ba796522-af29-11e7-9597-763aeed34de2.png)
```
W 0:01:34.971843 {PPU[0x1000000] Thread (main_thread) [0x00bd191c]} sysPrxForUser: sys_ppu_thread_create(thread_id=*0xd0040348, entry=0x1185ec8, arg=0x1a25a5c, prio=2048, stacksize=0x4000, flags=0x1, threadname=“ExtraSpaceCheckThread”)
·W 0:01:34.971849 {PPU[0x1000000] Thread (main_thread) [0x00bd191c]} sys_ppu_thread: _sys_ppu_thread_create(thread_id=*0xd0040348, param=*0xd00402c0, arg=0x1a25a5c, unk=0x0, prio=2048, stacksize=0x4000, flags=0x1, threadname=“ExtraSpaceCheckThread”)
·W 0:01:34.971949 {PPU[0x1000007] Thread (ExtraSpaceCheckThread) [0x00bd2250]} cellSaveData: cellSaveDataAutoSave2(version=0, dirName=“BLES00229PROFILE”, errDialog=1, setBuf=*0x1a48ec0, funcStat=*0x1186670, funcFile=*0x1186650, container=0xffffffff, userdata=*0x0)
·! 0:01:34.972149 {PPU[0x1000007] Thread (ExtraSpaceCheckThread)} Thread time: 0.000000s (0.001040Gc); Faults: 0 [rsx:0, spu:0];
·W 0:01:34.975690 {PPU[0x1000000] Thread (main_thread) [0x00bd191c]} sysPrxForUser: sys_ppu_thread_create(thread_id=*0xd00403e8, entry=0x1185ec8, arg=0x1a25a5c, prio=2048, stacksize=0x4000, flags=0x1, threadname=“SizesOfAllSavesThread”)
·W 0:01:34.975696 {PPU[0x1000000] Thread (main_thread) [0x00bd191c]} sys_ppu_thread: _sys_ppu_thread_create(thread_id=*0xd00403e8, param=*0xd0040360, arg=0x1a25a5c, unk=0x0, prio=2048, stacksize=0x4000, flags=0x1, threadname=“SizesOfAllSavesThread”)
·W 0:01:34.975775 {PPU[0x1000007] Thread (SizesOfAllSavesThread) [0x00bd2250]} cellSaveData: cellSaveDataListAutoLoad(version=0, errDialog=1, setList=*0x1a48ee8, setBuf=*0x1a48ec0, funcFixed=*0x11865f8, funcStat=*0x11865e8, funcFile=*0x11865f0, container=0xffffffff, userdata=*0x0)
·! 0:01:34.976044 {PPU[0x1000007] Thread (SizesOfAllSavesThread)} Thread time: 0.000000s (0.001282Gc); Faults: 0 [rsx:0, spu:0];
·W 0:01:34.978701 {PPU[0x1000000] Thread (main_thread) [0x00bd191c]} sysPrxForUser: sys_ppu_thread_create(thread_id=*0xd00402b8, entry=0x1185ec8, arg=0x1a25a5c, prio=2048, stacksize=0x4000, flags=0x1, threadname=“FreeSpaceCheckThread”)
·W 0:01:34.978706 {PPU[0x1000000] Thread (main_thread) [0x00bd191c]} sys_ppu_thread: _sys_ppu_thread_create(thread_id=*0xd00402b8, param=*0xd0040230, arg=0x1a25a5c, unk=0x0, prio=2048, stacksize=0x4000, flags=0x1, threadname=“FreeSpaceCheckThread”)
·W 0:01:34.978778 {PPU[0x1000007] Thread (FreeSpaceCheckThread) [0x00bd2250]} cellSaveData: cellSaveDataAutoSave2(version=0, dirName=“BLES00229PROFILE”, errDialog=1, setBuf=*0x1a48ec0, funcStat=*0x1186670, funcFile=*0x1186650, container=0xffffffff, userdata=*0x0)
·! 0:01:34.978924 {PPU[0x1000007] Thread (FreeSpaceCheckThread)} Thread time: 0.000000s (0.000774Gc); Faults: 0 [rsx:0, spu:0];
·W 0:01:34.980215 {PPU[0x1000000] Thread (main_thread) [0x00bd191c]} sysPrxForUser: sys_ppu_thread_create(thread_id=*0xd00402f8, entry=0x1185ec8, arg=0x1a25a5c, prio=2048, stacksize=0x4000, flags=0x1, threadname=“SaveThread”)
·W 0:01:34.980219 {PPU[0x1000000] Thread (main_thread) [0x00bd191c]} sys_ppu_thread: _sys_ppu_thread_create(thread_id=*0xd00402f8, param=*0xd0040270, arg=0x1a25a5c, unk=0x0, prio=2048, stacksize=0x4000, flags=0x1, threadname=“SaveThread”)
·W 0:01:34.980290 {PPU[0x1000007] Thread (SaveThread) [0x00bd2250]} cellSaveData: cellSaveDataAutoSave2(version=0, dirName=“BLES00229PROFILE”, errDialog=1, setBuf=*0x1a48ec0, funcStat=*0x1186670, funcFile=*0x1186650, container=0xffffffff, userdata=*0x0)
·! 0:01:34.983154 {PPU[0x1000007] Thread (SaveThread)} Thread time: 0.000000s (0.004982Gc); Faults: 0 [rsx:0, spu:0];
·W 0:01:34.984718 {PPU[0x1000000] Thread (main_thread) [0x00bd191c]} sysPrxForUser: sys_ppu_thread_create(thread_id=*0xd0040348, entry=0x1185ec8, arg=0x1a25a5c, prio=2048, stacksize=0x4000, flags=0x1, threadname=“ExtraSpaceCheckThread”)
·W 0:01:34.984724 {PPU[0x1000000] Thread (main_thread) [0x00bd191c]} sys_ppu_thread: _sys_ppu_thread_create(thread_id=*0xd0040348, param=*0xd00402c0, arg=0x1a25a5c, unk=0x0, prio=2048, stacksize=0x4000, flags=0x1, threadname=“ExtraSpaceCheckThread”)
·W 0:01:34.984843 {PPU[0x1000007] Thread (ExtraSpaceCheckThread) [0x00bd2250]} cellSaveData: cellSaveDataAutoSave2(version=0, dirName=“BLES00229PROFILE”, errDialog=1, setBuf=*0x1a48ec0, funcStat=*0x1186670, funcFile=*0x1186650, container=0xffffffff, userdata=*0x0)
·! 0:01:34.985047 {PPU[0x1000007] Thread (ExtraSpaceCheckThread)} Thread time: 0.000000s (0.001086Gc); Faults: 0 [rsx:0, spu:0];
```

[Log GTA IV 12-10-17.gz](https://github.com/RPCS3/rpcs3/files/1378341/Log.GTA.IV.12-10-17.gz)
[BLES00229PROFILE.zip](https://github.com/RPCS3/rpcs3/files/1378355/BLES00229PROFILE.zip)

**Comment by MarioSonic2987** [comment]
The regression is caused by PR #3005. Using this [build](https://ci.appveyor.com/project/MarioSonic2987/rpcs3/build/0.0.3-23/artifacts), which reverts that PR, fixes this issue.

**Comment by MarioSonic2987** [comment]
Fixed by PR #3918.
![](https://user-images.githubusercontent.com/20027681/33781267-27a063ec-dc54-11e7-9dca-cd108eb247ff.png)
I'll close this issue when it's merged.

**Comment by MarioSonic2987** [comment]
#3918 is merged. Closing this issue.
